src/librustc_llvm/ffi.rs

```diff
@@ -617,6 +617,8 @@ extern "C" {
     pub fn LLVMConstIntGetSExtValue(ConstantVal: ValueRef) -> c_longlong;
     pub fn LLVMRustConstInt128Get(ConstantVal: ValueRef, SExt: bool,
                                   high: *mut u64, low: *mut u64) -> bool;
+    pub fn LLVMRustIsConstantFP(ConstantVal: ValueRef) -> bool;
+    pub fn LLVMRustConstFloatGetBits(ConstantVal: ValueRef) -> u64;
```

Comment:
Does a bitcast + the existing methods for getting the value of an integer not work?

Reply:
No, they do, I just didn't realize that at the time I wrote this. Will update.